CMakeLists.txt

```diff
@@ -15,12 +15,6 @@ FetchContent_Declare(SFML
     GIT_TAG 2.6.x)
 FetchContent_MakeAvailable(SFML)
 
-
-
-add_executable(GameOfLife ${SOURCES})
-target_link_libraries(GameOfLife PRIVATE sfml-graphics)
-target_compile_features(GameOfLife PRIVATE cxx_std_20)
-
 if(WIN32)
```

Comment:
I'm not sure what this does, but if it's related to the executable it should be moved into that CMakeList.txt